huaweicloud/resource_huaweicloud_cbr_policy_test.go

```diff
@@ -0,0 +1,102 @@
+package huaweicloud
+
+import (
+	"fmt"
+	"log"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+
+	"github.com/huaweicloud/golangsdk/openstack/cbr/v3/policies"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+)
+
+func TestAccCBRV3Policy_basic(t *testing.T) {
+	var asPolicy policies.Policy
+	rName := fmt.Sprintf("tf-acc-test-%s", acctest.RandString(5))
+	resourceName := "huaweicloud_cbr_policy.test"
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckASV1PolicyDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testCBRV3Policy_basic(rName),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckCBRV3PolicyExists(resourceName, &asPolicy),
+					resource.TestCheckResourceAttr(resourceName, "name", rName),
+					resource.TestCheckResourceAttr(resourceName, "enabled", "true"),
+				),
+			},
```

Comment:
the update steps should be added,

Reply:
thanks, the update steps and import steps are supported.